### PR TITLE
Style: fix mobile responsiveness

### PR DESCRIPTION
**Pull request by anyanime**
- Center h1 text
- Make form responsive on mobile

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -9,11 +9,11 @@ function App() {
         className="flex flex-col justify-center items-center my-8"
       >
         <img src={logo} className="App-logo" alt="logo" width={150} />
-        <h1 className="text-3xl my-4">Volunteer Application Form</h1>
+        <h1 className="text-3xl my-4 text-center">Volunteer Application Form</h1>
```
Comment (stbensonimoh) [inline src/App.js:12]: Great work for catching this!

File: src/index.css [hunk 2]
```diff
@@ -27,7 +27,7 @@ form label {
 }
 
 form .column:first-child {
-  @apply pr-4;
+  @apply sm:pr-4;
```
Comment (stbensonimoh) [inline src/index.css:30]: Sweet 👍🏽

**Review by stbensonimoh — APPROVED** [review]
Great work! I'm proud of you.

File: src/index.css (hunk 2 above)
Comment (anyanime) [reply src/index.css:30]: Thank you very much. It's been great working with you.

File: src/App.js (hunk 1 above)
Comment (anyanime) [reply src/App.js:12]: Thank you very much.

**Pull request merged**